resources/netregexes.ts

```diff
@@ -31,37 +27,70 @@ const keysThatRequireTranslation = [
   'line',
 ];
 
+type ParseHelperField<
+  T extends TriggerTypes,
+  T2 extends NetFieldsReverse[T],
```

Comment:
If there's only one `T` then `T` is (sometimes) ok on its own.  If you have two, then please give them real names for readability (`Type` and `Fields`, maybe?)